benchmark/url/whatwg-url-to-and-from-path.js

```diff
@@ -3,28 +3,50 @@ const common = require('../common.js');
 const { fileURLToPath, pathToFileURL } = require('node:url');
 const isWindows = process.platform === 'win32';
 
-const bench = common.createBenchmark(main, {
-  input: isWindows ? [
-    'file:///c/',
-  ] : [
-    'file:///dev/null',
-    'file:///dev/null?key=param&bool',
-    'file:///dev/null?key=param&bool#hash',
-  ],
-  method: isWindows ? [
-    'fileURLToPath',
-  ] : [
-    'fileURLToPath',
-    'pathToFileURL',
-  ],
-  n: [5e6],
-});
+const inputs = isWindows ? [
+  'C:\\foo',
+  'C:\\Program Files\\Music\\Web Sys\\main.html?REQUEST=RADIO',
+  '\\\\nas\\My Docs\\File.doc',
```

Comment:
Let's also add a "long" version for good measure
```suggestion
  '\\\\nas\\My Docs\\File.doc',
  '\\\\?\\UNC\\server\\share\\folder\\file.txt',
```